client/my-sites/site-settings/media-settings-performance.jsx

```diff
@@ -125,21 +126,24 @@ class MediaSettingsPerformance extends Component {
 	}
 
 	renderVideoUpgradeNudge() {
-		const { isVideoPressAvailable, translate } = this.props;
+		const { isVideoPressAvailable, siteSlug, translate } = this.props;
 
 		return (
-			! shouldShowOfferResetFlow() &&
 			! isVideoPressAvailable && (
 				<UpsellNudge
+					title={ translate( 'Get unlimited video hosting' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'Unlimited Video Hosting' )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/improve/video-hosting-upsell-copy)